server/controllers/CommentController.js

```diff
@@ -2,18 +2,31 @@ const db = require("../models/index");
 const Comment = db.Comment;
 
 exports.get = async (req, res) => {
-  try {
```

Comment:
We don't have to delete this right? Can just comment out for now if we are using sample comments